src/pyhf/contrib/viz/brazil.py

```diff
@@ -1,9 +1,67 @@
 """Brazil Band Plots."""
 import numpy as np
+from matplotlib.container import Container
 
 
-def plot_results(ax, mutests, tests, test_size=0.05):
+class BrazilBandContainer(Container):
+    r"""
+    Container for the :obj:`matplotlib.artist` s created in a
```

Comment:
Not sure how to fix this space between the end of `matplotlib.artist` and the start of the `s`. If I remove it Sphinx freaks out. Thoughts @kratsg? Or just leave it.

Reply:
"Container for the :obj:`matplotlib.artist` objects created in a "..